argus/src/main/java/com/moldedbits/argus/ArgusSessionManager.java

```diff
@@ -24,7 +24,7 @@
      * @return True if a user is logged in, false otherwise
      */
     boolean isLoggedIn() {
-        return false;
+        return ArgusState.valueOf(argusStorage.getString(KEY_ARGUS_STATE, null)).equals(ArgusState.SIGNED_IN);
```

Comment:
I've initialized the value of KEY_ARGUS_STATE in ArgusStorage to SIGNED_OUT in ArgusSessionManager Constructor